Clarify password hash synchronization in upgrade docs (#7198)
EnterpriseDB/docs: product_docs/docs/pgd/6.4/upgrades/upgrading_major_rolling.mdx

### Confirm cluster version
Confirm the updated cluster version is version 6001 by running `bdr.group_raft_details`.

### Synchronize password hashes

> From any of the upgrades nodes, run the following query to ensure that password hashes are the same across all nodes for each user. This is required before applications switch to Connection Manager.

Reviewer: ```suggestion
From any upgrade node, run the following query to ensure password hashes are consistent across all nodes for each user. This is required before applications switch to Connection Manager.
```
